app/assets/javascripts/app/project/project_sidebar.js

```diff
@@ -7,7 +7,8 @@ var baseSidebar = {
 
   selectReward: function(event){
     var url = this.$(event.currentTarget).data('new_contribution_url');
-    this.navigate(url);
+    var acceptContributions = this.$(event.currentTarget).data('accept_contributions');
```

Comment:
Avoid having underscores on data attributes, your attribute should be named `data-accept-contributions` and then you'll be able to get it with `.data('acceptContributions')`